source/user-manual/agents/remote-upgrading/create-custom-wpk-automatically.rst

```diff
@@ -0,0 +1,102 @@
+.. Copyright (C) 2019 Wazuh, Inc.
+
+.. _create-custom-wpk-automatically:
+
+Automated custom WPK packages creation
+=======================================
+
+Wazuh provides an automated way of building WPK packages using docker so there is no need for any other dependency.
+
+To create a WPK package this way only docker and a X509 certificate are needed, in the case of Windows WPK a msi package is needed.
```

Comment:
This sentence doesn't make sense. In addition, there are several typos: `docker`=> `Docker`, `msi` => `MSI`

Reply:
Done in d3a58a1, 473937b, ff99d41